client/state/jetpack-connect/actions.js

```diff
@@ -420,5 +422,22 @@ export default {
 				} );
 			} );
 		};
+	},
+	selectPlanInAdvance( planSlug, site ) {
+		return ( dispatch ) => {
+			dispatch( {
+				type: JETPACK_CONNECT_SELECT_PLAN_IN_ADVANCE,
+				plan: planSlug,
+				site: site
+			} );
+		};
+	},
+	completeFlow( site ) {
+		return ( dispatch ) => {
+			dispatch( {
+				type: JETPACK_CONNECT_COMPLETE_FLOW,
+				site
+			} );
+		};
```

Comment:
We could use tests for the newly introduced actions. Could be added in a subsequent PR though.